http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstallSpec.scala

```diff
@@ -66,7 +66,9 @@ class CromwellRunnerAppInstallSpec extends BaseAppInstallSpec {
       s"postgres.dbnames.cromwellMetadata=$cromwellMetadataAzureDbName," +
       s"ecm.baseUri=https://externalcreds.dsde-dev.broadinstitute.org," +
       s"sam.baseUri=https://sam.test.org:443," +
-      s"sam.acrPullActionIdentityResourceId=spend-profile"
+      s"sam.acrPullActionIdentityResourceId=spend-profile," +
+      "bard.baseUri=https://terra-bard-dev.appspot.com," +
+      "bard.enabled=true"
```

Comment:
This should be set to false, because the test will read the config from reference.conf, same for the other test file below